README.md

```diff
@@ -3,9 +3,10 @@
 [![Build Status](https://travis-ci.org/cfpb/cfgov-refresh.png?branch=flapjack)](https://travis-ci.org/cfpb/cfgov-refresh?branch=flapjack)
 [![Code Climate](https://codeclimate.com/github/cfpb/cfgov-refresh.png?branch=flapjack)](https://codeclimate.com/github/cfpb/cfgov-refresh?branch=flapjack)
 
-The in-progress redesign of the [consumerfinance.gov](http://consumerfinance.gov) website.
+The redesign of the [www.consumerfinance.gov](http://www.consumerfinance.gov) website.
```

Comment:
I'll defer to @schaferjh but in my edit I was thinking it was still a redesign, but was just no longer in-progress—e.g. it had happened.